src/Histogram.ts

```diff
@@ -17,10 +17,15 @@ type HistogramData = {
 export default class Histogram {
   // no more than 2^32 pixels of any one intensity in the data!?!?!
   private bins: Uint32Array;
+  /** Min value in the original raw data. */
   private min: number;
+  /** Max value in the original raw data. */
   private max: number;
+  /** Size of each histogram bin in the scale of the original raw data. */
```

Comment:
I think restating "original raw data" is unnecessary here in the above comments. You could just say "data".  A Histogram is being given data values and the histogram is a collection of statistics about those data values.